deepfence_worker/cronjobs/neo4j.go

```diff
@@ -214,12 +217,14 @@ func CleanUpDB(ctx context.Context, task *asynq.Task) error {
 	if _, err = session.Run(`
 		MATCH (n:ContainerImage)
 		WHERE n.active = false
-		AND (NOT exists((n) <-[:SCANNED]-())
+		AND ((NOT exists((n) <-[:SCANNED]-()) 
+		AND n.updated_at < TIMESTAMP() - $delete_threshold_ms)
 		OR n.updated_at < TIMESTAMP()-$old_time_ms)
```

Comment:
What values for `$delete_threshold_ms` and `old_time_ms`?

Reply:
```
$delete_threshold_ms -> 1 hour
$old_time_ms -> This is set using the timeout setting for config inactive_delete_scan_result
Minimum is 1 day. Default is 30 days (in case not configured)
```